### PR TITLE
Host utilization

### PR DESCRIPTION
**Pull request by christopherhans**

### CODE REVIEW

File: prometheus-exporters/vrops-exporter/alerts/host.alerts [hunk 1]
```diff
@@ -198,3 +198,29 @@ groups:
     annotations:
       description: "vSphere High Availability (HA) has detected a possible host failure for `{{ $labels.hostsystem }}`. ({{ $labels.vcenter }})"
       summary: "vSphere High Availability (HA) has detected a possible host failure for `{{ $labels.hostsystem }}`. ({{ $labels.vcenter }})."
+
+  - alert: HostMemoryUtilizationHigh
+    expr: vrops_hostsystem_memory_usage_percentage > 90
```
Comment (richardtief) [inline prometheus-exporters/vrops-exporter/alerts/host.alerts:203]:
```suggestion
    expr: vrops_hostsystem_memory_usage_percentage{vccluster=~".*management.*"} > 90
```

File: prometheus-exporters/vrops-exporter/alerts/host.alerts [hunk 2]
```diff
@@ -198,3 +198,29 @@ groups:
...
+      tier: vmware
+      service: compute
+      context: "{{ $labels.hostsystem }} memory"
+      meta: "Memory utilization of host {{ $labels.hostsystem }} is above 90%. ({{ $labels.vcenter }}, {{ $labels.vccluster }})"
+    annotations:
+      description: "Memory utilization of host {{ $labels.hostsystem }} is above 90%. ({{ $labels.vcenter }}, {{ $labels.vccluster }})"
+      summary: "Memory utilization of host {{ $labels.hostsystem }} is above 90%. ({{ $labels.vcenter }}, {{ $labels.vccluster }})"
+
+  - alert: HostCPUUtilizationHigh
+    expr: vrops_hostsystem_cpu_usage_average_percentage > 90
```
Comment (richardtief) [inline prometheus-exporters/vrops-exporter/alerts/host.alerts:216]:
```suggestion
    expr: vrops_hostsystem_cpu_usage_average_percentage{vccluster=~".*management.*"} > 90
```

**Review by richardtief — CHANGES_REQUESTED** [review]
Thank you for bringing this in. But we only need warnings for aPods.

**Comment by christopherhans** [comment]
Thanks, wasn't clear from my notes.

**Review by richardtief — APPROVED** [review]
thanks!

**Pull request merged**